Components/SimplifiedGrid/simplified-grid.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Simplified Grid</title>
+    <link href="/CSS/simplified-grid.css" rel="stylesheet" type="text/css" />
```

Comment:
You can add a description in your meta tag as well like author details, page description